storage/spi/src/main/kotlin/StorageEntry.kt

```diff
@@ -39,21 +39,24 @@ data class StorageEntry(
     val data: InputStream,
 
     /** The content type associated with this entry if any. */
-    val contentType: String?
+    val contentType: String?,
```

Comment:
Unrelated to this commit, but now that I'm seeing this, could the meaning of a `null` `contentType` be documented here?

Reply:
I assume that simply means that the content type is unknown which has to be supported as the ORT reporter API does not force reporter implementations to provide the content type of the generated reports. But maybe @oheger-bosch could provide more details here.